fdbserver/storageserver.actor.cpp

```diff
@@ -3452,7 +3452,7 @@ ACTOR Future<Void> storageServerCore( StorageServer* self, StorageServerInterfac
 		double elapsedTime = loopTopTime - lastLoopTopTime;
 		if( elapsedTime > 0.050 ) {
 			if (deterministicRandom()->random01() < 0.01)
-				TraceEvent(SevWarn, "SlowSSLoopx100", self->thisServerID).detail("Elapsed", elapsedTime);
+				TraceEvent(SevWarn, "SlowSSLoopx100", self->thisServerID).suppressFor(30.0).detail("Elapsed", elapsedTime);
```

Comment:
This is both sampling and suppressing now.

Reply:
Yes, I understood that. 
Even with the 1% sampling, `SlowSSLoopx100 ` is still recorded in trace for over 100 times in one test case.

Reply:
Then maybe it indicates the threshold value for `SlowSSLoopx100` needs to larger than `0.050` ?

Reply:
The test case is ` -r simulation --logsize 1024MiB -f foundationdb/tests/slow/ddbalance.txt -b on -s 680105759`

Reply:
I don't think both should be necessary. With the suppression, it would only log once every 30 seconds. If the event is logging very frequently, the sampling isn't going to have much effect anyway (i.e. if it logs 100 times per second, then you're still going to log this event roughly every 30 seconds).